src/console_window.cpp

```diff
@@ -184,11 +193,13 @@ void ConsoleWindow::clearAll()
 {
   db_->clear();
   node_list_model_->clear();
+  db_proxy_->clearSearchFailure();//resets failed search variables, VCM 27 April 2017
```

Comment:
Comments at the end of a line should have two spaces before the `//` (and a single space after the `//`).